cranelift-codegen/meta/src/shared/instructions.rs

```diff
@@ -85,6 +85,15 @@ pub fn define(
         TypeSetBuilder::new().ints(32..64).build(),
     );
 
+    let Ref = &TypeVar::new(
+        "Ref",
+        "A scalar reference type",
+        TypeSetBuilder::new()
+            .refs(Interval::All)
+            .simd_lanes(Interval::All)
```

Comment:
I think this should be removed, otherwise this allows vector of reference types.